db/external_sst_file_ingestion_job.cc

```diff
@@ -250,6 +255,13 @@ void ExternalSstFileIngestionJob::UpdateStats() {
   uint64_t total_keys = 0;
   uint64_t total_l0_files = 0;
   uint64_t total_time = env_->NowMicros() - job_start_time_;
+
+  auto stream = event_logger_->Log();
```

Comment:
Maybe specify the type of `stream` explicitly? We prefer to use `auto` when the type is easily known from the context.